[🐞]  Error while deploying Qwik project with Cloudflare Pages adapter

### Which component is affected?

Qwik Runtime

### Describe the bug

I am trying to deploy a Qwik project using the Cloudflare Pages adapter, but I encountered an error while running npm run deploy. After selecting the project, I received the following error message:

✘ [ERROR] Could not resolve "..serverentry.cloudflare-pages"

    _worker.js:1:22:
      1 │ ...{ fetch } from "..\server\entry.cloudflare-pages"; export defaul...
        ╵                   ~~~~~~~~~~

  You can mark the path "..serverentry.cloudflare-pages" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Build failed with 1 error:

  _worker.js:1:22: ERROR: Could not resolve "..serverentry.cloudflare-pages"

### Reproduction

https://stackblitz.com/edit/qwik-starter-rqzdwk?file=README.md

### Steps to reproduce

1. Set up a Qwik project with the Cloudflare Pages adapter.
2. Run npm run deploy.
3. Select the project.
4. Expected Result

The project should be successfully deployed without any errors.

Actual Result

The deployment process fails with the error message mentioned above.

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (20) x64 12th Gen Intel(R) Core(TM) i9-12900H
    Memory: 15.41 GB / 31.70 GB
  Binaries:
    Node: 16.15.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files\nodejs\yarn.CMD
    npm: 8.19.2 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.1555.0), Chromium (112.0.1722.68)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @builder.io/qwik: 1.0.0 => 1.0.0
    @builder.io/qwik-city: ^1.0.0 => 1.0.0
    undici: 5.22.0 => 5.22.0
    vite: 4.3.3 => 4.3.3
```


### Additional Information

Please let me know if there are any known workarounds or any additional information I can provide to help troubleshoot this issue. Thank you!

--- a/packages/qwik-city/adapters/cloudflare-pages/vite/index.ts
+++ b/packages/qwik-city/adapters/cloudflare-pages/vite/index.ts
@@ -2,6 +2,7 @@ import type { StaticGenerateRenderOptions } from '@builder.io/qwik-city/static';
 import { type ServerAdapterOptions, viteAdapter } from '../../shared/vite';
 import fs from 'node:fs';
 import { join, relative } from 'node:path';
+import { normalizePathSlash } from '../../../utils/fs';
 
 /**
  * @public
@@ -55,7 +56,7 @@ export function cloudflarePagesAdapter(opts: CloudflarePagesAdapterOptions = {})
         const importPath = relative(clientOutDir, join(serverOutDir, 'entry.cloudflare-pages'));
         await fs.promises.writeFile(
           workerJsPath,
-          `import { fetch } from "${importPath}"; export default { fetch };`
+          `import { fetch } from "${normalizePathSlash(importPath)}"; export default { fetch };`
         );
       }
     },

--- a/packages/qwik-city/utils/fs.ts
+++ b/packages/qwik-city/utils/fs.ts
@@ -67,8 +67,10 @@ export function removeExtension(fileName: string) {
 }
 
 export function normalizePath(path: string) {
-  path = normalize(path);
+  return normalizePathSlash(normalize(path));
+}
 
+export function normalizePathSlash(path: string) {
   // MIT https://github.com/sindresorhus/slash/blob/main/license
   // Convert Windows backslash paths to slash paths: foo\\bar ➔ foo/bar
   const isExtendedLengthPath = /^\\\\\?\\/.test(path);

